client/src/pages/OnBoarding/smallComponents/AvailableDays.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import {
+  Checkbox,
+  Grid,
+  FormControlLabel,
+  makeStyles,
+  Typography,
+} from "@material-ui/core";
+
+const week = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+
+const AvailableDays = (props) => {
+  const classes = useStyles();
+
+  const days = props.days;
+  const setDays = props.setDays;
+
+  const selectDay = (event) => {
+    const newDays = { ...days, [event.target.name]: event.target.checked };
+    setDays(newDays);
+  };
+
+  return (
+    <Grid
+      container
+      justify="flex-start"
+      wrap="nowrap"
+      className={classes.daysGrid}
+    >
+      {week.map((item, i) => {
+        return (
+          <FormControlLabel
+            key={i}
+            label={
+              <Typography variant="caption" className={classes.labelText}>
+                {item}
+              </Typography>
+            }
+            labelPlacement="bottom"
+            className={classes.label}
+            control={
+              <Checkbox
+                className={classes.eachDay}
+                checked={days[item] || false}
+                onChange={selectDay}
+                name={item}
+                color="primary"
+              />
+            }
+          />
+        );
+      })}
+    </Grid>
+  );
+};
+
+const useStyles = makeStyles((theme) => ({
+  daysGrid: {
+    margin: "0",
+  },
+  eachDay: {
+    margin: "0",
+  },
+  label: {
```

Comment:
You should add some padding and/or include a different set width so that the text has room to breath but the boxes are consistent

Reply:
Yes I did check it out but it looks quite different from what I see on calendly. Time picker seems to only allow the option to display a clock pop up but in calendly they use a list of available time options. Either approach is okay with me. Should I switch to time picker?

Reply:
I think you can just stick with what you have in that case. One note though is to adjust the spacing for it because the -- looks a bit off center